document/cloud-client/set_endpoint.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START documentai_set_endpoint]
+from google.cloud import documentai_v1beta2 as documentai
+
+
+def set_endpoint(project_id='YOUR_PROJECT_ID',
+                 input_uri='gs://cloud-samples-data/documentai/invoice.pdf'):
+    """Process a single document with the Document AI API, including
+    text extraction and entity extraction."""
+
+    client = documentai.DocumentUnderstandingServiceClient(
+        client_options={"api_endpoint": "eu-documentai.googleapis.com"})
+
```

Comment:
Do they want the full snippet or just the part that changes the endpoint? 

In the past we've only done this:
```suggestion
def set_endpoint(project_id='YOUR_PROJECT_ID',
                 input_uri='gs://cloud-samples-data/documentai/invoice.pdf'):
    """Process a single document with the Document AI API, including
    text extraction and entity extraction."""
    
    # [START documentai_set_endpoint]
    from google.cloud import documentai_v1beta2 as documentai

    client = documentai.DocumentUnderstandingServiceClient(
        client_options={"api_endpoint": "eu-documentai.googleapis.com"})
    # [END documentai_set_endpoint]

```

Reply:
Done.